package text2;

public class ExceptionExample {
	public static void main(String[] args) {
		int a = 4;
		int b = 0;
		int c = a / b;
		System.out.println("cの値は" + c);
		System.out.println("処理を正常に終了します");
	}
}
